CHANGELOG.md

```diff
@@ -19,15 +26,23 @@ be found.
 + Add `--add-host` for docker build [#30383](https://github.com/docker/docker/pull/30383)
 + Add `.CreatedAt` placeholder for `docker network ls --format` [#29900](https://github.com/docker/docker/pull/29900)
 * Update order of `--secret-rm` and `--secret-add` [#29802](https://github.com/docker/docker/pull/29802)
-* Fix use of `**/` in `.dockerignore` [#29043](https://github.com/docker/docker/pull/29043)
 + Add `--filter enabled=true` for `docker plugin ls` [#28627](https://github.com/docker/docker/pull/28627)
 + Add `--format` to `docker service ls` [#28199](https://github.com/docker/docker/pull/28199)
 + Add `publish` and `expose` filter for `docker ps --filter` [#27557](https://github.com/docker/docker/pull/27557)
 * Support multiple service IDs on `docker service ps` [#25234](https://github.com/docker/docker/pull/25234)
 + Allow swarm join with `--availability=drain` [#24993](https://github.com/docker/docker/pull/24993)
-* Removal of the email from `docker login` [#26868](https://github.com/docker/docker/pull/26868)
++ Allow `plugin` management commands to address plugins by ID instead of name [#29222](https://github.com/docker/docker/pull/29222)
```

Comment:
I removed this one already, as @anusha-ragunathan pointed out, the feature was already here, this PR only adds a test